README.md

```diff
@@ -130,13 +130,13 @@ Before you use `terraform-provider-cockroach` you must [install Terraform](https
         - `<database name>` is the name that will be used for the database created within the cluster. This database is in addition to defaultdb which is created by default.  
         - `<SQL user name>` is the name of the SQL user you want to create.  
         - `<SQL user password>` is the password for the SQL user you want to create.  
-        - `<cloud provider>` is the cloud infrastructure provider. Possible values are `GCP` or `AWS` or `AZURE` (limited access).  
+        - `<cloud provider>` is the cloud infrastructure provider. Possible values are `GCP` or `AWS` or `AZURE`.  
         - `<cloud provider region>` is the region code or codes for the cloud infrastructure provider. For multi-region clusters, separate each region with a comma.  
         - `<number of nodes>` is the number of nodes in each region. Cockroach Labs recommends at least 3 nodes per region, and the same number of nodes in each region for multi-region clusters.  
+        - `<number of VCPUs per node>` is the number of virtual CPUS assigned to each node.  This number will dictate which machine type is used for your cluster nodes. Recommendations for choosing this value can be found [here](https://www.cockroachlabs.com/docs/cockroachcloud/create-your-cluster#step-5-configure-cluster-capacity).  
         - `<storage in GiB>` is the amount of storage specified in GiB.  
-        - `<cloud provider machine type>` is the machine type for the cloud infrastructure provider.  
-        - `<allow list name>` is the name for the IP allow list. Use a descriptive name to identify the IP allow list.  
-        - `<allow list CIDR IP>` is the Classless Inter-Domain Routing (CIDR) IP address base.  
+        - `<allow list name>` is the name for the IP allow list. Use a descriptive name to identify the IP allow list. (i.e. "allow all" or "home network")  
+        - `<allow list CIDR IP>` is the Classless Inter-Domain Routing (CIDR) IP address base. (i.e. 123.123.123.123)  
```

Comment:
NIT: would adding a mask be more representative of CIDR notation? E.g. 198.51.100.14/24

Reply:
Our allowlist resource asks for the ip portion and the prefix length separately.  Unfortunately we also use the name cidr_mask to refer to the prefix length.   I've tried to clear that up with some documentation changes.  I wouldn't be opposed to deprecating the existing eventually and using a single string field for the full cidr notation as a single string.